package.json

```diff
@@ -214,7 +214,7 @@
     "balena-semver": "^2.3.0",
     "balena-settings-client": "^4.0.7",
     "balena-settings-storage": "^7.0.0",
-    "balena-sync": "^11.0.2",
+    "balena-sync": "11.0.3-lucianbuzzo-typescript-32279716b412d16d5eadacf35d18f6b10996ea28",
```

Comment:
balena-io-modules/balena-sync/pull/167 was merged, now this line can be updated.

Reply:
I've updated the dep and will now move this PR out of draft